src/App.jsx

```diff
@@ -14,57 +15,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [chosenItem, setChosenItem] = useState('Jam');
+  const [isSelected, setIsSelectedState] = useState(true);
+  const [selectedRow, setSelectedRow] = useState('Jam');
 
```

Comment:
Recommend making the clearSelection function separate to improve code readability.

```
 const clearSelection = () => {
    setChosenItem('');
    setIsSelected(false);
    setSelectedRow('');
};
```